### ISSUE TITLE
Cannot use spinner-directive within an element that is display:none at application startup

### ISSUE THREAD

**Issue by bpatra** [ngOfficeUIFabric/ng-officeuifabric#199]
The size of the spinner is computed by method `createCirclesAndArrange`,the size of the circle is based on 

```
    let width: number = this.$element[0].clientWidth;
    let height: number = this.$element[0].clientHeight;
```

but if the `<uif-spinner uif-size="large"></uif-spinner>` is placed within an html element that is not displayed at application startup: `clientWidth` and `clientHeight` is zero and are not recomputed after that.

**Comment by bpatra** [comment]
Trying to reproduce in isolated plunker http://plnkr.co/edit/DaJQvfz19FOBtOw7vR1N (not ready yet)

**Comment by bpatra** [comment]
Looks to be fixed with release 0.4.0
